cirq/google/optimizers/convert_to_sqrt_iswap.py

```diff
@@ -101,6 +101,9 @@ def convert(self, op: 'cirq.Operation') -> List['cirq.Operation']:
         return a
 
     def optimization_at(self, circuit, index, op):
+        if isinstance(op.gate, ops.MatrixGate) and len(op.qubits) == 1:
+            return None
```

Comment:
Won't this leave Matrix gates alone?  Don't we need to transform them to PhasedXZ's or something to serialize them correctly?

Reply:
see the test, but single qubit gates are compiled explicitly as the second step in `optimized_for_sycamore`.

Reply:
I've verified that this change does indeed let you run single qubit matrix gates through quantum engine

Reply:
The test just tests that the number of moments is 2.  Can we add an extra line to verify that it serializes correctly too?

Reply:
added